src/components/CardView/CardView.js

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react';
+
+const CardView = (props) => {
+  const { title, text, imageUrl, imageAlt, buttonUrl, buttonText } = props;
+
+  let imageEle;
+  if (imageUrl) {
+    imageEle = <img className="card-img-top" src={imageUrl} alt={imageAlt} />;
+  }
```

Comment:
Breaks the line length rule :( (especially on the button. I need to put that on multiple lines with the extra param